pandas/tests/io/json/test_pandas.py

```diff
@@ -511,6 +511,43 @@ def test_blocks_compat_GH9037(self):
                            by_blocks=True,
                            check_exact=True)
 
+    def test_frame_nonprintable_bytes(self):
+        # GH14256: failing column caused segfaults, if it is not the last one
+
+        class BinaryThing(object):
+
+            def __init__(self, hexed):
+                self.hexed = hexed
+                if compat.PY2:
+                    self.binary = hexed.decode('hex')
+                else:
+                    self.binary = bytes.fromhex(hexed)
+
+            def __str__(self):
+                return self.hexed
+
+        hexed = '574b4454ba8c5eb4f98a8f45'
+        exc_type = OverflowError
+        binthing = BinaryThing(hexed)
+        df_printable = DataFrame({'A': [binthing.hexed]})
```

Comment:
before each sub-section, can put a comment on what you are testing; and blank lines between sub-sections